packages/backend/src/open_payments/receiver/errors.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  errorToMessage as incomingPaymentErrorToMessage,
+  errorToCode as incomingPaymentErrorToCode,
+  isIncomingPaymentError,
+  IncomingPaymentError
+} from '../payment/incoming/errors'
+import {
+  errorToMessage as remoteIncomingPaymentErrorToMessage,
+  errorToCode as remoteIncomingPaymentErrorToCode,
+  RemoteIncomingPaymentError
+} from '../payment/incoming_remote/errors'
+
+export type ReceiverError = IncomingPaymentError | RemoteIncomingPaymentError
+export const ReceiverError = {
+  ...IncomingPaymentError,
+  ...RemoteIncomingPaymentError
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+export const isReceiverError = (o: any): o is ReceiverError =>
+  Object.values(ReceiverError).includes(o)
+
+export const receiverErrorToCode = (error: ReceiverError): number =>
+  isIncomingPaymentError(error)
+    ? incomingPaymentErrorToCode[error]
+    : remoteIncomingPaymentErrorToCode[error]
+
+export const receiverErrorToMessage = (error: ReceiverError): string =>
+  isIncomingPaymentError(error)
+    ? incomingPaymentErrorToMessage[error]
+    : remoteIncomingPaymentErrorToMessage[error]
```

Comment:
I could have also just spread the two `errorToMessage`/`errorToCode` objects into a new `receiver` `errorToMessage`/`errorToCode` object, but thought this was easier to follow